src/Control/Distributed/Spark/RDD.hs

```diff
@@ -119,6 +158,27 @@ aggregate seqOp combOp zero rdd = do
   res :: JObject <- call rdd "aggregate" [coerce jzero, coerce jseqOp, coerce jcombOp]
   reify (unsafeCast res)
 
+-- | A version of aggregate implemented in terms of 'mapPartitions'.
+aggregate'
```

Comment:
Open to anything given a good technical reason to do so. But is there a reason to *not* map our `aggregate` to Spark's `aggregate` (despite doing so being a tad surprising for users) and instead make it stand for our own implementation based on some other function?